CONTRIBUTING.md

```diff
@@ -39,9 +39,11 @@ Once the Hermit environment is activated you can type the following to start a
 hot-reloading ftl agent:
 
 ```
-$ ftl devel ./examples/echo ./examples/time
+$ ftl dev ./examples/echo ./examples/time
```

Comment:
Just a heads up, I think currently the `ftl dev` command might not work with multiple paths. But it will soon! I think that will be the next CLI change I work on.